benchmarks/FSharpBenchmarks/BenchmarkDotNet.Artifacts/results/IcedTasks.Benchmarks.AsyncBenchmarks-report-github.md

```diff
@@ -1,40 +1,40 @@
-|          NonAsyncBinds_cancellableTask_bindTask | NonAsyncBinds |  23,018.4 μs |    174.25 μs |    145.50 μs |  23,031.6 μs |  2.00 |    0.06 | 11375.0000 |       - |      - |  92,969 KB |
-|                   NonAsyncBinds_cancellableTask | NonAsyncBinds |  23,183.0 μs |    371.83 μs |    397.86 μs |  23,260.6 μs |  2.00 |    0.06 | 11375.0000 |       - |      - |  92,969 KB |
-|                                                 |               |              |              |              |              |       |         |            |         |        |            |
-|                          AsyncBinds_CSharpTasks |    AsyncBinds |  17,388.5 μs |    603.48 μs |  1,779.38 μs |  17,583.3 μs |  1.00 |    0.00 |          - |       - |      - |     109 KB |
-|                                  AsyncBinds_ply |    AsyncBinds |  22,273.6 μs |    525.30 μs |  1,507.19 μs |  22,066.7 μs |  1.30 |    0.17 |    62.5000 |       - |      - |     656 KB |
-|                                AsyncBinds_async |    AsyncBinds |  88,393.7 μs |  1,737.46 μs |  2,378.26 μs |  87,425.4 μs |  5.71 |    0.56 |  1000.0000 |       - |      - |   8,375 KB |
-|                                 AsyncBinds_task |    AsyncBinds |  19,577.5 μs |    442.81 μs |  1,277.62 μs |  19,482.5 μs |  1.14 |    0.13 |          - |       - |      - |     188 KB |
-|                    AsyncBinds_coldTask_bindTask |    AsyncBinds |  18,582.4 μs |    623.38 μs |  1,828.28 μs |  18,193.2 μs |  1.09 |    0.18 |    31.2500 |       - |      - |     305 KB |
-|                AsyncBinds_coldTask_bindColdTask |    AsyncBinds |  19,229.0 μs |    375.91 μs |    742.00 μs |  19,219.6 μs |  1.19 |    0.14 |    31.2500 |       - |      - |     305 KB |
-|             AsyncBinds_cancellableTask_bindTask |    AsyncBinds |  19,890.7 μs |    392.33 μs |    687.13 μs |  19,941.4 μs |  1.24 |    0.13 |    31.2500 |       - |      - |     320 KB |
-|         AsyncBinds_cancellableTask_bindColdTask |    AsyncBinds |  19,412.4 μs |    387.11 μs |    857.81 μs |  19,334.6 μs |  1.19 |    0.12 |    31.2500 |       - |      - |     320 KB |
-|  AsyncBinds_cancellableTask_bindCancellableTask |    AsyncBinds |  20,201.2 μs |    400.21 μs |    852.88 μs |  20,049.0 μs |  1.25 |    0.15 |    31.2500 |       - |      - |     320 KB |
+|                                          Method |    Categories |         Mean |        Error |       StdDev |       Median |  Ratio | RatioSD |       Gen0 |    Allocated | Alloc Ratio |
+|------------------------------------------------ |-------------- |-------------:|-------------:|-------------:|-------------:|-------:|--------:|-----------:|-------------:|------------:|
+|                          AsyncBinds_CSharpTasks |    AsyncBinds |   3,329.6 μs |     28.27 μs |     26.45 μs |   3,334.1 μs |   1.00 |    0.00 |     3.9063 |    109.39 KB |        1.00 |
```

Comment:
This went from 17μs to 3μs? Is this due to the changed setup or can "old vs new" be compared? It's a pretty amazing perf gain!

Reply:
I wouldn't read too much into these as this has changed machines.